beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -793,12 +810,15 @@ func TestProposer_ProposeBlock_OK(t *testing.T) {
 
 			c := &mock.ChainService{Root: bsRoot[:], State: beaconState}
 			db := dbutil.SetupDB(t)
+			var bb *builderTest.MockBuilderService
+			if tt.builderService != nil {
```

Comment:
do we need these lines of code instead of just using tt.buildService() where bb is and return nil?